pkg/controller/core.oam.dev/v1alpha2/application/apply_test.go

```diff
@@ -240,51 +156,6 @@ var _ = Describe("Test Application apply", func() {
 	})
 })
 
-var _ = Describe("Test applyHelmModuleResources", func() {
```

Comment:
what will the test logic be after refactor?

Reply:
helm resources will be regarded as workload addon resources which are just a slice of unstructueds. The logic has been been tested in assemble/options_test.go